src/language-js/index.js

```diff
@@ -35,7 +35,7 @@ const languages = [
   createLanguage(require("linguist-languages/data/typescript"), {
     override: {
       since: "1.4.0",
-      parsers: ["typescript-eslint"],
+      parsers: ["typescript-estree"],
```

Comment:
Context: https://github.com/prettier/prettier/pull/5139#discussion_r219929107

It seems you forgot to revert this one.

Reply:
I think we can change this to simply `["typescript"]`